tests/bluetooth/bsim_bt/bsim_test_app/tests_scripts/basic_conn.sh

```diff
@@ -24,12 +24,13 @@ BOARD="${BOARD:-nrf52_bsim}"
 
 cd ${BSIM_OUT_PATH}/bin
 
-Execute ./bs_${BOARD}_samples_bluetooth_peripheral_prj_conf \
-  -v=${VERBOSITY_LEVEL} -s=${SIMULATION_ID} -d=0 -RealEncryption=0
+Execute ./bs_${BOARD}_tests_bluetooth_bsim_bt_bsim_test_app_prj_conf \
+  -v=${VERBOSITY_LEVEL} -s=${SIMULATION_ID} -d=0 -RealEncryption=0 \
+  -testid=peripheral -rs=6
```

Comment:
Seeding both devices with the same random seed seems a bit funny (same in the other testcase script below)

Reply:
No need to do funny stuff, I use a different seed now for the peripheral.